addons/mail/static/src/components/notification_list/notification_list_notification_group_tests.js

```diff
@@ -419,24 +419,21 @@ QUnit.test('different mail.channel are not grouped', async function (assert) {
     );
 });
 
-QUnit.test('multiple grouped notifications by document model, sorted by date desc', async function (assert) {
+QUnit.test('multiple grouped notifications by document model, sorted by the most recent message of each group', async function (assert) {
     assert.expect(9);
 
     this.data['mail.message'].records.push(
         // first message that is expected to have a failure
         {
-            date: moment.utc().format("YYYY-MM-DD HH:mm:ss"), // random date
-            id: 11, // random unique id, will be used to link failure to message
+            id: 12, // random unique id, will be used to link failure to message
             message_type: 'email', // message must be email (goal of the test)
             model: 'res.partner', // different model from second message
             res_id: 31, // random unique id, useful to link failure to message
             res_model_name: "Partner", // random related model name
         },
         // second message that is expected to have a failure
         {
-            // random date, later than first message
-            date: moment.utc().add(1, 'days').format("YYYY-MM-DD HH:mm:ss"),
-            id: 12, // random unique id, will be used to link failure to message
+            id: 11, // random unique id, will be used to link failure to message
```

Comment:
This diff still indicates a problem in code I think, test is expecting "Company" to be the first notification, and the first notification should be the most recent, which technically has the highest id (which was the case already before this diff).